evm-ext/src/native_evm_context.rs

```diff
@@ -33,6 +36,7 @@ pub struct HeaderForExecution {
 pub struct NativeEVMContext<'a> {
     pub resolver: &'a dyn MoveResolver<PartialVMError>,
     pub storage_trie: &'a dyn StorageTrieRepository,
+    pub transfers_log: &'a dyn EthTransferLog,
```

Comment:
nit: should it be `transfer_logs` instead of `transfers_log`?

Reply:
Changed in https://github.com/MovedNetwork/op-move/pull/319/commits/56a3eb19db003048aa820ecb849ed6d6fceee767